packages/timeline-state-resolver/src/conductor.ts

```diff
@@ -621,33 +647,43 @@ export class Conductor extends EventEmitter<ConductorEvents> {
 			if (this.devices.has(deviceId)) {
 				throw new Error(`Device "${deviceId}" already exists when creating device`)
 			}
+			throwIfAborted()
 			this.devices.set(deviceId, newDevice)
 
 			return newDevice
 		} catch (e) {
-			if (newDevice) {
-				try {
-					await newDevice.terminate()
-				} catch (e) {
-					this.emit('error', `Cleanup failed of aborted device "${newDevice.deviceId}": ${e}`)
-				}
-			}
+			await this.terminateUnwantedDevice(newDevice)
 			this.devices.delete(deviceId)
 			this.emit('error', 'conductor.createDevice', e)
 			return Promise.reject(e)
 		}
 	}
+	private async terminateUnwantedDevice(newDevice: DeviceContainer<DeviceOptionsBase<any>> | undefined) {
```

Comment:
Minor thing: A newline between the previous method and the one coming after this one would be much appreciated :)

Reply:
I added some empty lines